apps/frontend/src/app/app.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import { render } from '@testing-library/react';
```

Comment:
Nice choice) I also like this library, which I probably mentioned before)))

Reply:
This is not my choice 😄